core/flyout.js

```diff
@@ -130,6 +137,29 @@ Blockly.Flyout.prototype.MARGIN = 8;
 Blockly.Flyout.prototype.SCROLLBAR_PADDING = 2;
 
 /**
+ * Size of a checkbox next to a variable reporter.
+ * @type {number}
+ * @const
+ */
+Blockly.Flyout.prototype.CHECKBOX_SIZE = 20;
```

Comment:
Can this be set and calculated in CSS, or is it necessary for layout that it's defined in the JS? It comes to mind that we could want to have the checkbox a different size in touch environments from desktop environments...I wonder how we should deal with that if it comes up.